src/macaron/config/defaults.ini

```diff
@@ -475,3 +475,18 @@ hostname = registry.npmjs.org
 # The attestation REST API.
 attestation_endpoint = -/npm/v1/attestations
 request_timeout = 20
+
+# Configuration options for selecting the checks to run.
+# Both the exclude and include are defined as lists of string:
+#   - The exclude list is used to specify the checks that will not run.
+#   - The include list is used to specify the checks will run. Checks are not specified by "include" will be excluded.
+# The final list of checks to run will always be "include" minus "exclude".
+# information on the effect of include/exclude to the check dependencies.
+# Each element of a list is either the full ID of a check (e.g mcn_build_as_code_1) or a glob pattern
```

Comment:
It is not clear how the user can provide the list of checks. I think one example would be helpful.

Reply:
I have added a few examples in https://github.com/oracle/macaron/pull/254/commits/5efae2c587d19d1e172a7eb749417f62eeb0ae96